packages/blade/src/components/Text/Decisions/Decisions.md

```diff
@@ -0,0 +1,87 @@
+The text component is categorised into multiple parts
```

Comment:
Could restructure the folder a bit better. 

/spec/decisions.md
/specs/accessibility.md - in Future.

Reply:
decisions will have all sorts of decisions, be it API, be it Accessibility, or anything else. otherwise, it's challenging to discover these files. since it's mostly meta-information i would prefer to keep it under one file. if this had been a component or any other file the recommendation would have been valid.